tests/js/launch-test-harness.js

```diff
@@ -3,8 +3,20 @@
 var fluid = require("infusion");
```

Comment:
These 3 files are not tests but test infrastructure which is to be used by others - I would put them in src/test